cmd/ragengine/main.go

```diff
@@ -57,7 +52,6 @@ func init() {
 	utilruntime.Must(kaitoutils.AwsSchemeBuilder.AddToScheme(scheme))
 
 	//+kubebuilder:scaffold:scheme
-	klog.InitFlags(nil)
```

Comment:
I find we use klog in many other places.... Can you confirm again whether we can remove this line...